sklearn/model_selection/tests/test_validation.py

```diff
@@ -1118,14 +1123,12 @@ def test_learning_curve_batch_and_incremental_learning_are_equal():
     estimator = PassiveAggressiveClassifier(max_iter=1, tol=None,
                                             shuffle=False)
 
-    train_sizes_inc, train_scores_inc, test_scores_inc = \
-        learning_curve(
-            estimator, X, y, train_sizes=train_sizes,
-            cv=3, exploit_incremental_learning=True)
+    train_sizes_inc, train_scores_inc, test_scores_inc = learning_curve(
```

Comment:
Please avoid unrelated changes.

Reply:
Sorry, artifact of previous changes, reverted back in 299d5d3